pkg/collector/collector.go

```diff
@@ -192,3 +216,12 @@ func (wb WalletBalanceCollector) Collect(ch chan<- prometheus.Metric) {
 
 	log.Debug("Stop collecting", zap.String("metric", walletBalanceMetricName))
 }
+
+func getDiscordIDs(ops []config.Operator) string {
+	var ids []string
+	for _, op := range ops {
+		ids = append(ids, op.Discord.ID)
```

Comment:
I think we should append `op.Discord.ID` to `ids` only if it is not empty string.

Reply:
Added a check and test cases for this